modules/kubernetes-addons/aws-pca-issuer/main.tf

```diff
@@ -0,0 +1,23 @@
+module "cert_manager" {
+  source            = "./../cert-manager"
+  helm_config       = var.cert_manager_helm_config
+  manage_via_gitops = var.manage_via_gitops
+  addon_context     = var.addon_context
+}
```

Comment:
cert manager should be deployed from the consumer example . This add-on should only install AWS PCA Issuer

Reply:
hi @vara-bonthu - Any pointers on how customers add dependency between two modules ? 

For example, aws-pca-issuer depends on cert-manager. Customer enable `cert-manager` by `enable_cert_manager                 = true` and pca issuer by `enable_aws_pca_issuer               = true` . Which is why, i added the dependency aws-pca-issuer itself, which requires `cert-manager` as a pre-req.

Reply:
We need to design a proper dependency mechanism between addons. @anshrma at this point please document the dependency and update your usage example to show `enable_cert_manager = true` is needed.